pydft\_qmmm.utils.exceptions.PyDFTQMMMException
===============================================

.. currentmodule:: pydft_qmmm.utils.exceptions

.. autoclass:: PyDFTQMMMException

   

   
   

   
    
   